README.md

```diff
@@ -67,6 +67,116 @@ The `Consuming` field provides information about the `Percentage` of messages co
 }
 ```
 
+## Metrics
+
+In order to check how your Apache Kafka cluster is behaving, the Canary provides the following metrics on the corresponding HTTP endpoint.
```

Comment:
What is the metrics endpoint, I don't think it's mentioned in the README.

Reply:
The Endpoints section talks about endpoints so it mentions the `/metrics` one.

Reply:
But what port is it on?

Reply:
8080 ... I will add it to the description.